test/ui-test/lightspeedRoleGenTest.ts

```diff
@@ -158,5 +157,88 @@ describe("Verify Role generation reset button works as expected", function () {
     await getWebviewByLocator(
       By.xpath("//*[text()='What do you want the role to accomplish?']"),
     );
+
+    await workbenchExecuteCommand("View: Close All Editor Groups");
+  });
+
+  it("Role generation (outline reset, cancel)", async function () {
+    await VSBrowser.instance.openResources(
```

Comment:
This is a lot of dup code. Could you prepare a function called `goToPage2()` or something like this?